src/coreclr/src/.nuget/Microsoft.NET.Sdk.IL/targets/Microsoft.NET.Sdk.IL.targets

```diff
@@ -144,6 +145,37 @@ Copyright (c) .NET Foundation. All rights reserved.
     <CallTarget Targets="$(TargetsTriggeredByCompilation)" Condition="'$(TargetsTriggeredByCompilation)' != ''"/>
   </Target>
 
+  <Target Name="GenerateVersionFile">
+   <PropertyGroup>
+      <VersionFilePath Condition="'$(VersionFilePath)' == ''">$(IncludePath)version.h</VersionFilePath>
```

Comment:
This file is auto-generated by the build. Should the path point to an intermediate folder by default instead?